### DISCUSSION TITLE
Prowler 3.0 and PostgresSQL

### DISCUSSION THREAD

**Discussion by NSi1974** [prowler-cloud/prowler#1566] [General]
Hi !

Congrats for this new version, which seems awesome :)
I've got a quick question : can I still implement the Database providers connector, as for the v2 versions ?
If not, do you have any plan for it or is it no longer an option ?

Thank you all !

Nicolas

**Comment by toniblyx** [comment]
Thanks @NSi1974!

We are not sure if to implement again the Postgresql connector en v3 since we don't see if it has been really used. In any case it looks like it is not very hard to implement with [psycopg2](https://pypi.org/project/psycopg2/) and take advantage of the new CSV or Json output formats.

Long story short, we will decide over January about this, let's see what other community members have to say.
